### PR TITLE
DruidBeam: Fix exception at TRACE logging level.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

File: core/src/main/scala/com/metamx/tranquility/druid/DruidBeam.scala [hunk 1]
```diff
@@ -88,7 +88,7 @@ class DruidBeam[A](
       if (log.isTraceEnabled) {
         log.trace(
           "Sending %,d events to task[%s], firehose[%s]: %s",
```
Comment (rasahner) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeam.scala]: %,d really?

Comment (gianm) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeam.scala]: Is that bad?

Comment (rasahner) [inline core/src/main/scala/com/metamx/tranquility/druid/DruidBeam.scala]: I guess it's a legal format I just wasn't familiar with.

**Comment by pjain1** [comment]
:+1:

**Comment by rasahner** [comment]
+1

**Pull request merged**